src/seq/seqrecord.jl

```diff
@@ -31,6 +36,11 @@ function Base.getindex(seqrec::SeqRecord, r::UnitRange)
 end
 
 
+function Base.copy{T <: SeqRecord}(seqrec::T)
```

Comment:
I think I ended up doing a lot of this, which is inconsistent with our style. I was getting pretty annoyed with importing things. It's starting to feel like C/C++ where you have to make sure declarations and definitions in different files match. I want to rethink this style choice, but maybe this PR isn't the right place to do that, so I can change these.